### PR TITLE
Fix adding Adblock list

### LINKED ISSUE #1801: Can't add Adblock list
My guess that is because of the **apostrophe** in the file name.
Steps to reproduce:
Go to [this](http://www.kiboke-studio.hr/i-dont-care-about-cookies/) page
Click on **Add to Adblock Plus** link
Click :ok: on confirmation dialogue
The result is the following:
capture du 2015-12-10 21 20 30

No problem with this with _V1.8 Qt4_

### PR DESCRIPTION
**Pull request by innermous**
Closes #1801 .

### CODE REVIEW

File: src/lib/adblock/adblockmanager.cpp [hunk 1]
```diff
@@ -154,10 +154,14 @@ bool AdBlockManager::addSubscriptionFromUrl(const QUrl &url)
 
     for (int i = 0; i < queryItems.count(); ++i) {
         QPair<QString, QString> pair = queryItems.at(i);
-        if (pair.first == QL1S("location"))
-            subscriptionUrl = pair.second;
-        else if (pair.first == QL1S("title"))
-            subscriptionTitle = pair.second;
+        if (pair.first == QL1S("location")) {
+            QString temp = pair.second;
+            subscriptionUrl = QUrl(temp).path();
```
Comment (nowrep) [inline src/lib/adblock/adblockmanager.cpp]: That doesn't look right. What about `QUrl::fromPercentEncoding`?

Comment (innermous) [inline src/lib/adblock/adblockmanager.cpp]: That works correctly, not sure QUrl::fromPercentEncoding will.

Comment (nowrep) [inline src/lib/adblock/adblockmanager.cpp]: It works, but maybe the correct fix is just `QUrlQuery::queryItems(QUrl::FullyDecoded))`

Comment (innermous) [inline src/lib/adblock/adblockmanager.cpp]: Well, if it is, it's up to you then :)

Comment (nowrep) [inline src/lib/adblock/adblockmanager.cpp]: You can update the pull request if you want :)

Comment (innermous) [inline src/lib/adblock/adblockmanager.cpp]: I'd check correctness of that first, if you haven't yet.

Comment (innermous) [inline src/lib/adblock/adblockmanager.cpp]: You are right, that was so obvious.

Comment (nowrep) [inline src/lib/adblock/adblockmanager.cpp]: No, it wasn't. I needed to check how it is used in other place to see the error ;)

Comment (innermous) [inline src/lib/adblock/adblockmanager.cpp]: So update PR?

Comment (nowrep) [inline src/lib/adblock/adblockmanager.cpp]: Yes, please.

**Pull request merged**